setup.py

```diff
@@ -22,8 +22,9 @@
     }
 }
 
+
 setup(name='l2rpn_baselines',
-      version='0.2.1',
+      version=__version__,
```

Comment:
Why not use `l2rpn_baseline.__version__` directly instead of duplicating

Reply:
i'm not sure pip install will work with that, but that's a thing we probably need to test indeed ;-)

Reply:
i'm not sure pip install will work with that, but that's a thing we probably need to test indeed ;-)